services/QuillLMS/app/models/translated_text.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: translated_texts
+#
+#  id                 :bigint           not null, primary key
+#  locale             :string           not null
+#  translation        :text
+#  created_at         :datetime         not null
+#  updated_at         :datetime         not null
+#  english_text_id    :integer          not null
+#  translation_job_id :string           not null
+#
+class TranslatedText < ApplicationRecord
+  belongs_to :english_text
+  scope :pending_translation, -> { where(translation: nil) }
+
+  def self.fetch_pending!
```

Comment:
`fetch` is often used for pulling records, consider `translate_pending!` to make the action clearer (I was surprised to see this translate records).

Reply:
It pulls the records from Gengo, so I think `fetch` is the right thing here. This method was moved to `app/services/gengo/save_translated_text.rb`